src/emailjs-imap-client.js

```diff
@@ -128,7 +128,7 @@
             this.client.onerror = this._onError.bind(this);
         }).catch((err) => {
             this.logger.error('Could not connect to server', err);
-            this.close(); // we don't really care whether this works or not
+            this.close(err); // we don't really care whether this works or not
```

Comment:
The comment is explaining why there is no .then nor .catch for the resolution of this promise call. We have encountered an error and we try to close but that may not work since the error may be caused by a broken connection.